source/posts/2018-09-11-post1.html.markdown

```diff
@@ -1,120 +1,108 @@
-
-        <p>My thoughts are that having fluent coding skills is not necessary for a designer to have, but it is <em>absolutely</em>beneficial if a designer either already possesses that knowledge on their belt, is willing to actually invest the time and energy into learning it, or seeks professional development because design is a constant path of learning, improvement, and growth. After researching and reading a few articles, here are some conclusions about the benefits and drawbacks of
-          being a designer who codes:</p>
-
-      <br />
-
-<section class="blogpost__post--column">
-      <h5><strong>BENEFITS</strong></h5>
-        <hr />
-      <h4><strong>UX Research and UI Design Process</strong></h4>
-        <ul class='blogposts__ul--indent'>
-          <li>Understanding the medium with which you create helps in understanding knowing the limitations of the current system, as well as user pain points and possible solutions for those problems.</li>
-
-          <li>Completely avoids the risk of blurring the lines between art and design. Design serves a very specific purpose and solves problems, whereas art is intended to be consumed and make you question, contemplate, and feel.</li>
-        </ul>
-
-      <h4><strong>Now you’re speaking the same language. Literally.</strong></h4>
-        <ul class='blogposts__ul--indent'>
-          <li>Fosters a stronger community, creates a deeper understanding between designers and developers, and offers a more effective way of pairing.</li>
-
-          <li>Developers&#8217; understanding of the language and its limitations can greatly influence design decisions.</li>
-
-          <li>Newfound knowledge can help you discover fresh approaches to common problems, develop new ways of thinking, and learn to consider your work more objectively.</li>
-        </ul>
-
-      <h4><strong>Stay fresh, stay current</strong></h4>
-        <ul class='blogposts__ul--indent'>
-            <li>Although is it not absolutely necessary that a designer learn to code, it is necessary for a designer tofully understand technologies and how they work.</li>
-
-            <li>Having a knowledge of code will lead to increased empathy for your teammates&#8217; respective challenges.</li>
-
-            <li>Learning and mastering new skills promotes innovation and creativity in itself.</li>
-
-            <li>Possessing fluent coding skills secures a stronger position in the job market.</li>
-        </ul>
-
-          <br />
```

Comment:
Awesome job getting rid of these <br> tags! It looks like you know never to use them. Way to go.